### PR TITLE
New intro and validation tables for builtins only.

### PR DESCRIPTION
**Pull request by Meggin**
Note that there are links to the new validation errors doc which isn't live yet in ampproject.org.

We will need to make the validation errors doc live before merging these tables, or else the links to that doc will be broken.

### CODE REVIEW

File: builtins/amp-ad.md [hunk 1]
```diff
@@ -24,12 +14,61 @@ See the License for the specific language governing permissions and
...
+    <th width="40%"><strong>Validation Error</strong></th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td width="40%"><a href="/docs/reference/validation_errors.html#mandatory-attribute-missing">MANDATORY_ATTR_MISSING</a></td>
+    <td>Error thrown when <code>type</code> attribute missing.</td>
+  </tr>
+  <tr>
+    <td width="40%"><a href="/docs/reference/validation_errors.html#disallowed-attribute">DISALLOWED_ATTR</a></td>
+    <td>Error thrown when tag includes disallowed <code>srcset</code> attribute.</td>
```
Comment (Gregable) [inline builtins/amp-ad.md]: DISALLOWED_ATTR can happen for any attribute, `srcset`, `style`, `lemur`, etc.

File: builtins/amp-video.md [hunk 2]
```diff
@@ -16,13 +16,51 @@ limitations under the License.
...
+(see also `amp-video` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
+    <th>Description</th>
+  </tr>
+    <tr>
+    <td width="40%"><a href="/docs/reference/validation_errors.html#mandatory-tag-ancestor-with-hint">MANDATORY_TAG_ANCESTOR_WITH_HINT</a></td>
+    <td>Error thrown if your AMP document uses <code>video</code> instead of <code>amp-video</code>. Error message: The tag <code>video</code> may only appear as a descendant of tag <code>noscript</code>. Did you mean <code>amp-video</code>?</td>
```
Comment (Gregable) [inline builtins/amp-video.md]: suggestion, optional. Maybe put <code></code> around the entire error message? Same below for amp-img.

File: builtins/amp-video.md [hunk 3]
```diff
@@ -16,13 +16,51 @@ limitations under the License.
...
+    <td><a href="https://github.com/ampproject/amphtml/blob/master/examples/everything.amp.html">everything.amp.html</a></td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-video` tag
+(see also `amp-video` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
```
Comment (Gregable) [inline builtins/amp-video.md]: Optional. Unsure if it's worth mentioning, but we'll also throw an INVALID_ATTRIBUTE_VALUE error if a handful of attributes have a value (they must be valueless): controls, autoplay, loop, muted.

**Comment by rudygalfi** [comment]
LGTM

File: builtins/amp-video.md (hunk 3 above)
Comment (erwinmombay) [inline builtins/amp-video.md]: spec should be fixed now. https://github.com/ampproject/amphtml/blob/master/extensions/amp-carousel/amp-carousel.md wording might a be a bit unclear, might need help on that @Meggin

**Pull request closed without merging**